src/panel/direction/DirectionInput.jsx

```diff
@@ -106,6 +107,7 @@ class DirectionInput extends React.Component {
               <input
                 ref={inputRef}
                 id={`direction-input_${pointType}`}
+                className={classnames({ valid: point !== null })}
```

Comment:
Just testing `point` for truthiness with `{ valid: point }` would be simpler and more robust, if for some reason it can become `undefined`. For example if the state management of the parent components change.

Reply:
How about:   !!point
Unnecessary?

Reply:
Unnecessary because classnames checks the truthiness of the argument, it doesn't need a real boolean. But if you find it clearer, no problem for me as it will behave correctly with `undefined`.